src/VideogameList.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { getVideogames } from './utils.js';
+import './VideogameList.css';
+
+class VideogameList extends Component {
+    state = { videogames: [] };
+    componentDidMount = async () => {
+        const data = await getVideogames();
+        this.setState({ videogames: data });
+    };
+
+    render() {
+        return (
+            <section className='games'>
+                {this.state.videogames.map((item) => (
+                    <div className='game'>
```

Comment:
Instead of returning a `<div>` here, I would have liked to see you use a list format instead. This is more semantic and accessible than simply using a `<div>` and best practice. 

If a list format doesn't work for you, another option within the section is to use the `<article>` tag, which is an isolated component within a section and covers both semantics and accessibility.